web/modules/custom/asset_release/asset_release.routing.yml

```diff
@@ -9,4 +9,17 @@ asset_release.solution_asset_release.add:
   options:
     parameters:
       rdf_entity:
-        type: entity:rdf_entity
\ No newline at end of file
+        type: entity:rdf_entity
+
+# Route to show an overview of releases belonging to a solution.
+asset_release.solution_asset_release.overview:
+  path: '/solution/{rdf_entity}/releases'
```

Comment:
We are not using the bundle as the first argument of the path. This should be /rdf_entity/{rdf_entity}/releases.
I don't think we should introduce new base of the arguments for a single query since the solution is also using the '/rdf_entity/'. This is for consistency reasons.